lib/kernel/src/erl_boot_server.erl

```diff
@@ -61,8 +61,8 @@ and `m:erl_prim_loader` in ERTS.
 -behaviour(gen_server).
 
 %% API functions.
--export([start/1, start_link/1, add_slave/1, delete_slave/1,
-	 add_subnet/2, delete_subnet/2,
+-export([start/1, start/2, start_link/1, start_link/2,
+	 add_slave/1, delete_slave/1, add_subnet/2, delete_subnet/2,
```

Comment:
Nitpick: since we're touching these lines anyway, let's make later additions easier while we're at it :)

```suggestion
-export([start/1, start/2,
         start_link/1, start_link/2,
         add_slave/1, delete_slave/1,
         add_subnet/2, delete_subnet/2,
```